validator/src/commands/contact_info/mod.rs

```diff
@@ -36,3 +50,43 @@ pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
         print!("{contact_info}");
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use {
+        super::*,
+        crate::commands::tests::{
+            verify_args_struct_by_command, verify_args_struct_by_command_is_error,
+        },
+    };
+
+    #[test]
+    fn verify_args_struct_by_command_contact_info_output_json() {
+        verify_args_struct_by_command(
+            command(&DefaultArgs::default()),
+            vec!["contact-info", "--output", "json"],
+            ContactInfoArg {
+                output: Some("json".to_string()),
+            },
+        );
+    }
+
+    #[test]
+    fn verify_args_struct_by_command_contact_info_output_json_compact() {
+        verify_args_struct_by_command(
+            command(&DefaultArgs::default()),
+            vec!["contact-info", "--output", "json-compact"],
+            ContactInfoArg {
+                output: Some("json-compact".to_string()),
+            },
+        );
+    }
+
+    #[test]
+    fn verify_args_struct_by_command_contact_info_output_invalid() {
+        verify_args_struct_by_command_is_error::<ContactInfoArg>(
+            command(&DefaultArgs::default()),
+            vec!["contact-info", "--output", "xxx"],
```

Comment:
nit: "xxx" ==> "invalid_output_type"

Pretty minor but is a bit of a self-documenting name

Reply:
e13f891bfa